ci/htcondor.sh

```diff
@@ -16,7 +16,7 @@ function jobqueue_before_install {
 
 function jobqueue_install {
     cd ./ci/htcondor
-    docker-compose exec -T submit /bin/bash -c "cd /dask-jobqueue; pip3 install -e .;chown -R submituser ."
+    docker-compose exec -T submit /bin/bash -c "cd /dask-jobqueue; pip3 install -e .[test]; chown -R submituser ."
```

Comment:
In the end does not really work because the wheel for cryptography is missing as well as a rust compiler to build it from source. I left it in because it still seemed a sound thing for me to do.

Reply:
Can you precise what it does?

Reply:
I am adding the optional test dependencies for HTCondor here, which actually fails, but does not matter due to the change in the test semantics. This might actually work at any given time: I have no clue why this wheel was not prebuild.